src/ui/map.js

```diff
@@ -2640,6 +2636,10 @@ class Map extends Camera {
 
         if (this._loaded && !this._fullyLoaded && !somethingDirty) {
             this._fullyLoaded = true;
+            if (this._isInitialLoad) {
```

Comment:
The _initialLoad boolean shouldn't be required because_fullyLoaded tracks the same state.